bigquery/noxfile.py

```diff
@@ -81,7 +81,7 @@ def system(session):
     session.install("--pre", "grpcio")
 
     # Install all test dependencies, then install local packages in place.
-    session.install("mock", "pytest")
+    session.install("mock", "pytest", "psutil")
```

Comment:
I didn't pin the version here, because `mock` and `pytest` are not pinned, either.